### PR TITLE
Create pt_BR.json

### PR DESCRIPTION
**Pull request by ManiaGamer**

### CODE REVIEW

File: pt_BR.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "localizedName": "Português (Brasil)",
+  "pluginVersion": "2.1.3",
+  "authors": "lenis, Plazmaz",
```
Comment (Plazmaz) [inline pt_BR.json]: You should change this to just your name.

File: pt_BR.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  "messageLogin": "Por favor use /login <senha>",
+  "messageRegister": "Por favor registre-se usando /register <senha>",
+  "errorRefreshProfile": "Sua conta foi modifica,por favor re-logue!",
+  "errorNotRegistered": "Usuário não está registrado!",
+  "kickAlreadyOnline": "Este jogador já está online!",
+  "kickUsernameChars": "Seu nome de usuário contém caracteres inválidos!",
+  "kickUsernameLength": "Seu nome de usuário é muito longo/curto! (%min% - %max% caracteres)",
+  "kickTimeOut": "O Tempo de login expirou"
+}
+//Please,give creditis for i?
```
Comment (Plazmaz) [inline pt_BR.json]: This should be removed.

**Comment by Plazmaz** [comment]
@ManiaGamer following up with you on this. If you'd like I can implement the changes in a new PR

**Comment by Tzzzt123** [comment]
@Plazmaz if u say something must be changed do it yourself that will be faster
and credit ManiaGamer yourself too
no offense

**Comment by Plazmaz** [comment]
@Tzzzt123 it was just a suggestion...

**Comment by Tzzzt123** [comment]
@Plazmaz oke but we need it to reach 20 translations...

**Comment by lenis0012** [comment]
@Tzzzt123 Merging a version with incorrect formatting (see last line) will break the system.
It should be fixed by the author

**Comment by Tzzzt123** [comment]
@lenis0012 But why should the author fix it? Why can't we do it?
Thanks for reply

**Comment by lenis0012** [comment]
@Tzzzt123 Well, it has been over a month without response.
So if you want, you can resubmit the PR with the requested fixes.

If you do, ill pull that one and those this PR.

**Comment by Tzzzt123** [comment]
@lenis0012 ok

**Pull request closed without merging**